internal/testutil/gpgkeys.go

```diff
@@ -0,0 +1,225 @@
+package testutil
+
+import (
+	"log"
+
+	"golang.org/x/crypto/openpgp/packet"
+
+	"github.com/canonical/chisel/internal/setup"
+)
+
+type Key struct {
+	ID                string
+	ArmoredPublicKey  string
+	ArmoredPrivateKey string
+	PublicKey         *packet.PublicKey
+	PrivateKey        *packet.PrivateKey
+}
+
+var gpgKeys = map[string]*Key{
+	"ubuntu-archive-key-2018": {
+		ID:               "871920D1991BC93C",
+		ArmoredPublicKey: ubuntuArchiveSignKey2018,
+	},
+	"ubuntu-archive-key-2012": {
+		ID:               "3B4FE6ACC0B21F32",
+		ArmoredPublicKey: ubuntuArchiveSignKey2012,
+	},
+	"test-key-1": {
+		ID:                "854BAF1AA9D76600",
+		ArmoredPublicKey:  testPublicKeyData,
+		ArmoredPrivateKey: testPrivateKeyData,
+	},
+	"test-key-2": {
+		ID:                "9568570379BF1F43",
+		ArmoredPublicKey:  testPublicKeyData2,
+		ArmoredPrivateKey: testPrivateKeyData2,
+	},
+}
+
+func init() {
+	for name, key := range gpgKeys {
+		if key.ArmoredPublicKey != "" {
+			pubKeys, privKeys, err := setup.DecodeKeys([]byte(key.ArmoredPublicKey))
+			if err != nil || len(privKeys) > 0 || len(pubKeys) != 1 || pubKeys[0].KeyIdString() != key.ID {
+				log.Panicf("invalid public key armored data: %s", name)
+			}
+			key.PublicKey = pubKeys[0]
+		}
+		if key.ArmoredPrivateKey != "" {
+			pubKeys, privKeys, err := setup.DecodeKeys([]byte(key.ArmoredPrivateKey))
+			if err != nil || len(pubKeys) > 0 || len(privKeys) != 1 || privKeys[0].KeyIdString() != key.ID {
+				log.Panicf("invalid private key armored data: %s", name)
+			}
+			key.PrivateKey = privKeys[0]
+		}
+	}
+}
+
+func GetGPGKey(name string) *Key {
```

Comment:
s/Get//, per typical Go conventions.

Reply:
I have also renamed it to `PGPKey` to make the naming consistent. PGP or openPGP for the standard and GPG for the concrete implementation.